Client/Frontend/Browser/Toolbars/UrlBar/TabLocationView.swift

```diff
@@ -396,6 +396,8 @@ extension TabLocationView: Themeable {
 
         readerModeButton.unselectedTintColor = theme.colors.tints.header
         readerModeButton.selectedTintColor = theme.colors.accent
+        
+        lockImageView.tintColor = theme.colors.tints.header
```

Comment:
I believe this should be changed to `appearanceTintColor` in order to support themes on iOS12, it's our own custom setter